#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_man_pages.dpatch by  <Hector Garcia <hector@debian.org>>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Clean up man pages

@DPATCH@
diff -urNad lirc-0.8.3~pre1~/doc/man/irexec.1 lirc-0.8.3~pre1/doc/man/irexec.1
--- lirc-0.8.3~pre1~/doc/man/irexec.1	2007-10-13 12:40:44.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/irexec.1	2007-10-30 10:14:05.000000000 -0400
@@ -29,7 +29,7 @@
 \fB\-n\fR \fB\-\-name\fR
 use this program name
 .SH OPTIONS
-If you add the --daemon option irexec will fork to
+If you add the \-\-daemon option irexec will fork to
 background. That way you can easily start irexec from an init script. In
 this case you should specify a config file on the command line as irexec
 won't be able to find your home directory. Potential uses are shutting
diff -urNad lirc-0.8.3~pre1~/doc/man/irrecord.1 lirc-0.8.3~pre1/doc/man/irrecord.1
--- lirc-0.8.3~pre1~/doc/man/irrecord.1	2007-10-13 12:40:46.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/irrecord.1	2007-10-30 10:14:05.000000000 -0400
@@ -14,7 +14,7 @@
 features of a remote control.
 
 If the program fails to recognize the protocol of the remote control you
-should use the --force option to at least create a config file in raw mode.
+should use the \-\-force option to at least create a config file in raw mode.
 
 If \fIfile\fR already exists and contains a valid config irrecord will use the
 protocol description found there and will only try to record the
diff -urNad lirc-0.8.3~pre1~/doc/man/irxevent.1 lirc-0.8.3~pre1/doc/man/irxevent.1
--- lirc-0.8.3~pre1~/doc/man/irxevent.1	2007-10-13 12:40:48.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/irxevent.1	2007-10-30 10:14:05.000000000 -0400
@@ -165,7 +165,7 @@
 .SH TROUBLESHOOTING
 
 If you have problems finding the coordinates for a button click you can try
-xev -id <window_id>. The window_id can be found using xwininfo. If xev and
+xev \-id <window_id>. The window_id can be found using xwininfo. If xev and
 xwininfo are not part of your distribution you can find them at a FTP server
 using the search engine at: http:\/\/ftpsearch.ntnu.no/ . xev also reports the
 names of key symbols like "Control_L" (your left control key) or "KP_Subtract"
@@ -187,7 +187,7 @@
 Yet another possibility is to start xterm like this:
 
 .RS 3
-xterm -xrm "XTerm.vt100.allowSendEvents: true"
+xterm \-xrm "XTerm.vt100.allowSendEvents: true"
 .RE
 
 xemacs will accept events if you set a built-in variable. The following was
diff -urNad lirc-0.8.3~pre1~/doc/man/lircd.8 lirc-0.8.3~pre1/doc/man/lircd.8
--- lirc-0.8.3~pre1~/doc/man/lircd.8	2007-10-13 12:40:46.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/lircd.8	2007-10-30 10:14:05.000000000 -0400
@@ -52,14 +52,14 @@
 \fB\-a\fR \fB\-\-allow\-simulate\fR
 accept SIMULATE command
 .SH OPTIONS
-The --permission option gives the file permission of /dev/lircd if it
+The \-\-permission option gives the file permission of /dev/lircd if it
 has to be created in octal representation. Read the documentation for
-chmod for further details. If no --permission option is given when the
+chmod for further details. If no \-\-permission option is given when the
 socket is initially created the default is to give all users read and
 write permissions (0666 in octal representation). If /dev/lircd already
 exists this option has no effect.
 
-With the --device option you can select the character device which lircd
+With the \-\-device option you can select the character device which lircd
 should read from. The default currently is /dev/lirc but it probably
 will change in future.
 
@@ -69,21 +69,21 @@
 the device name isn't fixed. \fISTRING\fR may contain the '*' and '?' 
 wildcards and '\\' to mark them as literal.
 
-With the --listen option you can let lircd listen for network
+With the \-\-listen option you can let lircd listen for network
 connections on the given port. The default port is 8765. No security
 checks are currently implemented.
 
-The --connect option allows you to connect to other lircd servers that
+The \-\-connect option allows you to connect to other lircd servers that
 provide a network socket at the given host and port number. The number
 of such connections is currently limited to 100.
 
-With the --output option you can select Unix domain socket, which lircd
+With the \-\-output option you can select Unix domain socket, which lircd
 will write remote key codes to. The default currently is /dev/lircd.
 
-With the --pidfile option you can select the lircd daemon pid file.
+With the \-\-pidfile option you can select the lircd daemon pid file.
 The default currently is /var/run/lircd.pid.
 
-With the --logfile option you can select the lircd daemon log file.
+With the \-\-logfile option you can select the lircd daemon log file.
 The default currently is /var/log/lircd. Note that this option will
 only be available if you compiled lircd without syslog support.
 
diff -urNad lirc-0.8.3~pre1~/doc/man/lircmd.8 lirc-0.8.3~pre1/doc/man/lircmd.8
--- lirc-0.8.3~pre1~/doc/man/lircmd.8	2007-10-13 12:40:46.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/lircmd.8	2007-10-30 10:14:05.000000000 -0400
@@ -21,7 +21,7 @@
 \fB\-n\fR \fB\-\-nodaemon\fR
 don't fork to background
 .SH OPTIONS
-If you provide the --nodaemon option lircmd won't fork to background.
+If you provide the \-\-nodaemon option lircmd won't fork to background.
 .SH FILES
 lircmd will use syslogd to output error messages. It depends on your
 system configuration where they will show up.
diff -urNad lirc-0.8.3~pre1~/doc/man/lircrcd.1 lirc-0.8.3~pre1/doc/man/lircrcd.1
--- lirc-0.8.3~pre1~/doc/man/lircrcd.1	2007-10-13 12:40:47.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/lircrcd.1	2007-10-30 10:14:05.000000000 -0400
@@ -32,15 +32,15 @@
 .SH OPTIONS
 .TP
 .BI \-p,\ \-\-premission= mode
-The --permission option gives the file permission of the Unix domain
+The \-\-permission option gives the file permission of the Unix domain
 socket lircrcd creates on startup in octal representation. Read the
-documentation for chmod for further details. If no --permission option
+documentation for chmod for further details. If no \-\-permission option
 is given when the socket is created the default is to give only the
 user owning the file read and write permissions (0600 in octal
 representation).
 .TP
 .BI \-o,\ \-\-output= socket
-With the --output option you can select the Unix domain socket, which
+With the \-\-output option you can select the Unix domain socket, which
 lircrcd will create. The default is to append a "d" character to the
 config filename given.
 .SH "SEE ALSO"
diff -urNad lirc-0.8.3~pre1~/doc/man/xmode2.1 lirc-0.8.3~pre1/doc/man/xmode2.1
--- lirc-0.8.3~pre1~/doc/man/xmode2.1	2007-10-13 12:40:45.000000000 -0400
+++ lirc-0.8.3~pre1/doc/man/xmode2.1	2007-10-30 10:14:05.000000000 -0400
@@ -15,7 +15,7 @@
 
 xmode2 is based on smode2 by Sinkovics Zoltan. It is just a conversion
 from svga to X with some basic support for resizing. The only
-differences are that the --mode option enables an alternative display
+differences are that the \-\-mode option enables an alternative display
 mode and that xmode2 can read and visualise mode2 output from stdin.
 .TP
 \fB\-h\fR \fB\-\-help\fR
